sdkproject/Assets/Mapbox/Unity/Map/AbstractMap.cs

```diff
@@ -126,7 +131,18 @@ protected virtual void Start()
 
 		void TileProvider_OnTileAdded(UnwrappedTileId tileId)
 		{
-			_mapVisualizer.LoadTile(tileId);
+			var utile = _mapVisualizer.LoadTile(tileId);
+			if (!_centerTileSet)
+			{
+				_centerTileSet = true;
+
+				// FIXME: this is never unregistered!
```

Comment:
This is never unregistered. If you try to move around when the tile is recycled, the map root transform will pop to the new height.